engine/src/main/java/org/hibernate/search/analyzer/definition/impl/LuceneTokenizerDefinitionContextImpl.java

```diff
@@ -17,6 +17,10 @@
  */
 public class LuceneTokenizerDefinitionContextImpl implements LuceneAnalysisDefinitionBuilder<TokenizerDef> {
 
+	private static final String PARAMS2 = "params";
```

Comment:
there is params variable present, used eclipse to refactor code.
params2 matches more closely to "params" value.